Analyzer/interface/CommonFunction.h

```diff
@@ -1505,23 +1505,17 @@ reco::DeDxData computedEdx(const int& run_number,
                            float* scaleFactors,
                            TH3* templateHisto = nullptr,
                            bool usePixel = false,
+                           bool useStrip = true,
                            bool useClusterCleaning = true,
-                           bool reverseProb = false,
                            bool useTruncated = false,
-                           std::unordered_map<unsigned int, float>* TrackerGains = nullptr,
```

Comment:
this is really not used anymore, even the current could doesnt do anything, it sets `prevAPV` and that's never used...

Reply:
Better way to deal with this is through a GT anyway

Reply:
I think it's to run on 2016 data. Idk if it's relevant to remove it.

Reply:
But @dapparu , the `prevAPV` is not consumed in the code. Also, UL dealt with the HIP issue, so we dont even need this for 2016